spacebench/api/dataverse.py

```diff
@@ -62,6 +63,22 @@ def __get_fileids_from_filenames(self):
             if filename in self.files.keys():
                 self.files[filename] = file["dataFile"]["id"]
 
+    def list_data_files(self, include_fileid=False):
+        """ Lists data files from Dataverse. """
+
+        files_list = self.dataset.json()[
+            'data']['latestVersion']['files']
```

Comment:
To ensure consistency, please choose either single or double quotes and stick with your choice. I recommend using double quotes.